data_structures/binary_tree/avl_tree.py

```diff
@@ -204,14 +204,16 @@ def del_node(root, data):
     if root is None:
         return root
     if get_height(root.get_right()) - get_height(root.get_left()) == 2:
-        if get_height(root.get_right().get_right()) > \
-           get_height(root.get_right().get_left()):
+        if get_height(root.get_right().get_right()) > get_height(
+            root.get_right().get_left()
+        ):
             root = left_rotation(root)
```

Comment:
```
right_height = get_height(root.get_right().get_right())
right_left_height = get_height(root.get_right().get_left())
if(right_height > right_left_height):
    root = left_rotation(root)
```
Now it's unreadable :cry:

Reply:
The first variable would be `right_right_height` which is no more readable than `get_height(root.get_right().get_right())`.

Reply:
@cclauss Could we refactor this in this pr? If not i will approve it without and open new with refactor.